### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by pierre**
See individual commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,54 +23,52 @@ The plugin needs a database. The latest version of the schema can be found [here
...
-
+* In addition to this, and depending on which information the templates require, other fields may be needed (e.g `address1`, `city`, ...)
 
 ## SMTP & Email Type Configuration
 
 ### Tenant Configuration
 
 Each tenant that requires the use of the plugin must be configured with the SMTP properties, and it can also specify the default set of emails that should be set. As indicated above, the plugin allows to be used to react to the following events: 
-`INVOICE_CREATION`, `UPCOMING_INVOICE`, `SUCCESSFUL_PAYMENT`, `FAILED_PAYMENT`, `PAYMENT_REFUND`, `SUBSCRIPTION_CANCELLATION_REQUESTED`, `SUBSCRIPTION_CANCELLATION_EFFECTIVE`:
+`INVOICE_CREATION`, `INVOICE_NOTIFICATION`, `INVOICE_PAYMENT_SUCCESS`, `INVOICE_PAYMENT_FAILED`, and `SUBSCRIPTION_CANCEL`.
```
Comment (sbrossie) [inline README.md]: So fir the cancelation, we have no way to differentiate  between requested and effective date ?

Comment (pierre) [reply README.md]: You must indeed subscribe to both events today: https://github.com/killbill/killbill-email-notifications-plugin/blob/0e2eae4fa67513468870f28e7e80dd35844e7df6/src/main/java/org/killbill/billing/plugin/notification/setup/EmailNotificationListener.java#L147-L149

We allow for configuration of 2 different templates: https://github.com/killbill/killbill-email-notifications-plugin/blob/0e2eae4fa67513468870f28e7e80dd35844e7df6/src/main/java/org/killbill/billing/plugin/notification/templates/TemplateType.java#L29-L30

but you cannot prevent requested cancellation emails to be sent out only for instance (if you don't configure a template, the default hardcoded one is used).

Something to review at some point...

**Comment by sbrossie** [comment]
Maybe file a GH so we don't forget?

**Comment by pierre** [comment]
Tracked by https://github.com/killbill/killbill-email-notifications-plugin/issues/33

**Pull request merged**